src/components/ui/sidebars/SidebarItems.tsx

```diff
@@ -19,7 +20,7 @@ export function IVSidebarItems({ items, level = 0 }: { items: SidebarItem[]; lev
     } else {
       return (
         // Value of item must be unique, such that triggers don't interfere with other items
-        <AccordionItem value={`item-${index}`} enableBorder={enableBorder}>
+        <AccordionItem key={`accordion-item-${id}`} value={`item-${index}`} enableBorder={enableBorder}>
```

Comment:
**(1)** Simply use `id`. Key is used as a unique identifier, and usually is not used for anything else.